opendrift/models/opendrift3D.py

```diff
@@ -338,6 +334,37 @@ def vertical_mixing(self):
  
         self.timer_end('main loop:updating elements:vertical mixing')
 
+    def plot_vertical_distribution_new(self, maxdepth=None, numtimes=5):
+        """Function to plot vertical distribution of particles"""
+        import matplotlib.pyplot as plt
+
+        fig = plt.figure()
+        ax = fig.add_subplot(111)
+        z = self.history['z']
+        if maxdepth is None:
+            maxdepth = z.min()
+        if maxdepth > 0:
+            maxdepth = -maxdepth  # negative z
+        dz=1
+        times = self.get_time_array()[0]
+        step = int(np.ceil(len(times)/numtimes))
+        ii = np.arange(1, len(times), step)
+        for i in ii:
+            ax.hist(z[:, i]+.01, bins=int(-maxdepth/dz),
+                    histtype='step',
+                    range=[maxdepth, 0], orientation='horizontal',
+                    label='hours: %s' % str(self.time_step.total_seconds()*i/3600))
+        ax.set_ylim([maxdepth, 0])
+        ax.set_xlabel('number of particles')
+        ax.set_ylabel('depth [m]')
+        #x_wind = self.history['x_wind'].T[tindex, :]
+        #y_wind = self.history['y_wind'].T[tindex, :]
+        #windspeed = np.mean(np.sqrt(x_wind**2 + y_wind**2))
+        #mainplot.set_title(str(self.get_time_array()[0][tindex]) +
+        #                   #'   Percent at surface: %.1f %' % percent_at_surface)
+        #                   '   Mean windspeed: %.1f m/s' % windspeed)
+        plt.legend()
+        plt.show()
```

Comment:
Would this show be problematic on non-desktop?

Reply:
Yes, possibly. Therefore I removed this new method in the previous commit.
On the other hand, I noticed that also "example_long...." scripts are now run on Circle-CI.
So seems like the filtering of these has disappeared? I guess that would explain the problems?

Reply:
hm, nothing has changed with that that I can see. They will be listed and documented, but not run.

Reply:
ok, I thought the listing meant that they were also run.
But then lets hope the last commit will solve the issue.

Reply:
I increased the timeout. And also the verbosity of example scripts run. This is on master, and currently building. You could rebase this PR and push again to get those changes, but might be worth waiting for the current run to finish first..?